src/rlgl.h

```diff
@@ -570,6 +571,7 @@ RLAPI Rectangle GetShapesTextureRec(void);                                // Get
 
 // Shader configuration functions
 RLAPI int GetShaderLocation(Shader shader, const char *uniformName);              // Get shader uniform location
+RLAPI int GetShaderAttribLocation(Shader shader, const char *attribName);         // Get shader attribute location
```

Comment:
I propose a change in name for  a bit more consistency with `GetShaderLocation()`:
```c 
RLAPI int GetShaderLocationAttrib(Shader shader, const char *attribName);
```

Reply:
Done